libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -697,10 +698,9 @@ void EthStratumClient::processReponse(Json::Value& responseObject)
 					cnote << "Subscribed to stratum server";
 
 					m_nextWorkDifficulty = 1;
-					jPrm = responseObject.get(((_rpcVer==1) ? "result":"params"), Json::Value::null);
-
-					if (!jPrm.empty() && jPrm.isArray()) {
-						std::string enonce = jPrm.get((Json::Value::ArrayIndex)1, "").asString();
+					
+					if (!jResult.empty() && jResult.isArray()) {
```

Comment:
Would this break responses marked as rpc v2 where additional array data about extra nonce is set into "params" member ?

Reply:
not sure where do you got the "params" part.
* old stratum used "result"
* json rpc 1 and 2 specify "result" for the response and "params" for the request.